modules/persistent-storage-csi-sc-multiple-none.adoc

```diff
@@ -16,35 +16,4 @@ There are two possible scenarios where PVCs can attempt to use a non-existent de
 
 * During installation, the installer creates a PVC requesting the default storage class, which has not yet been created.
 
-In the preceding scenarios, the PVCs remain in pending state indefinitely.
-
-{product-title} provides a feature to retroactively assign the default storage class to PVCs, so that they do not remain in the pending state. With this feature enabled, PVCs requesting the default storage class that are created when no default storage classes exists, remain in the pending state until a default storage class is created, or one of the existing storage classes is declared the default. As soon as the default storage class is created or declared, the PVC gets the new default storage class.
-
-:FeatureName: Retroactive default storage class assignment
-include::snippets/technology-preview.adoc[leveloffset=+1]
-
-=== Procedure
-
-To enable retroactive default storage class assignment:
-
-. Enable feature gates (see _Nodes_ → _Working with clusters_ → _Enabling features using feature gates_).
-+
-[IMPORTANT]
-====
-After turning on Technology Preview features using feature gates, they cannot be turned off. As a result, cluster upgrades are prevented.
-====
-+
-The following configuration example enables retroactive default storage class assignment, and all other Technology Preview features:
-+
-[source, yaml]
-----
-apiVersion: config.openshift.io/v1
-kind: FeatureGate
-metadata:
-  name: cluster
-spec:
-  featureSet: TechPreviewNoUpgrade <1>
-...
-----
-<1> Enables retroactive default storage class assignment.
-
+In the preceding scenarios, PVCs remain in the pending state indefinitely. To resolve this situation, create a default storage class or declare one of the existing storage classes as the default. As soon as the default storage class is created or declared, the PVC gets the new default storage class and moves out of the pending state.
```

Comment:
```suggestion
In the preceding scenarios, PVCs remain in the pending state indefinitely. To resolve this situation, create a default storage class or declare one of the existing storage classes as the default. As soon as the default storage class is created or declared, the PVCs get the new default storage class. If possible, the PVCs will eventually bind to statically or dynamically provisioned PVs as usual and move out of the pending state.
```
1) Use plural "PVCs" as all the the existing ones without storage class will be affected 

2) Assigning a storage class to pending PVCs does not necessarily grant moving out of the pending state by itself. It just allows for provisioning to continue as usual, which might not work and requires other (unrelated) troubleshooting.

3) Would it make sense to link to docs which describe "declaring storage class as default"? Like this one: https://github.com/openshift/openshift-docs/blob/875044019c418e282022f944682a798555d07b56/modules/dynamic-provisioning-change-default-class.adoc?plain=1#L63

Reply:
Done. The link you are suggesting is actually just below this module in the assemble for "Managing the default storage class", so I think that we are covered there.

Reply:
Cool, lgtm now.